### DISCUSSION TITLE
scikit-build Developer Meeting #18: June 7th, 2024

### DISCUSSION THREAD

**Discussion by thewtex** [scikit-build/scikit-build#1088] [Community meeting notes]
## Agenda 

- SciPy 2024

## Attending

- Matt McCormick
- Henry Scheiner
- Jean-Christophe Fillion-Robin

## Discussion

- [SciPy 2024 Paper](https://github.com/scipy-conference/scipy_proceedings/compare/2024...henryiii:scipy_proceedings:henry_schreiner_2024) in draft
- Windows FreeThreading support
    - FindPython
        - Fixed in CMake master

- next:
    - hatchling editable install support
    - [build section configuration consolidation](https://github.com/scikit-build/scikit-build-core/issues/747)
    - [invoking vcvars](https://github.com/scikit-build/cmake-FindVcvars)
    - Python build configuration state sharing improvements
        - https://github.com/python/cpython/pull/19740
        - https://peps.python.org/pep-0720/
        - https://ffy00.github.io/python-environments/
            - https://ffy00.github.io/python-environments/image-data.html#image-data--page-root
            - -> standard format for cross-compiling
    - `wheel` being integrated into `packaging` moving towards API support to generate a wheel
        - https://github.com/pypa/packaging/pull/805

### Recording and Transcript

- NA

**Comment by henryiii** [comment]
I remembered what I wanted to mention. I'm also planning on adding an "auto" choice for several values. `minimum-requires="auto"` would read the `scikit-build-core>=X` setting from `build-system.requires`. `cmake.version = "auto"` would try to read the CMake minimum from the `CMakeLists.txt`.
